docs/content/catalog.tex

```diff
@@ -0,0 +1,312 @@
+\subsection{Traders}
+
+Traders buy or sell in the virtual market. Different types of traders use different trading strategies. The same strategy may take different parameters thus perform differently. Currently TradingSimulation provides following indicators:\\
+
+\noindent
+\begin{tabularx}{\textwidth}{|c|X|}
+  \hline
+  Class & Description  \\
+  \hline
+  Arbitrageur & the \emph{arbitrageur} strategy \\
+  \hline
+  DoubleCrossOverTrader & the \emph{double cross over} strategy  \\
+  \hline
+  DoubleEnvelopeTrader & the \emph{double envelope} strategy  \\
+  \hline
+  SobiTrader & the \emph{static order book imbalance} strategy  \\
+  \hline
+  TransactionVwapTrader & the \emph{volume weighted average price} strategy  \\
+  \hline
+  SimpleTrader & a simple trader \\
+  \hline
+  SimpleFXtrader & a simple Forex trader \\
+  \hline
+\end{tabularx}
+
+\subsection{Fetchers}
+
+Fetchers in TradingSimulation play the role of feeding market data from external sources(internet or file system) into market simulator. Currently TradingSimulation provides following fetchers:\\
+
+\noindent
+\begin{tabularx}{\textwidth}{|l|X|}
+  \hline
+  Class & Description  \\
+  \hline
+  BtceOrderPullFetcher & the fetcher for BTC-e orders \\
+  \hline
+  BtceTransactionPullFetcher  & the fetcher for BTC-e transactions  \\
+  \hline
+  BitfinexOrderPullFetcher & the fetcher for Bitfinex orders  \\
+  \hline
+  BitfinexTransactionPullFetcher & the fetcher for Bitfinex transactions  \\
+  \hline
+  BitstampOrderPullFetcher & the fetcher for Bitstamp orders  \\
+  \hline
+  BitstampTransactionPullFetcher & the fetcher for Bitstamp transactions \\
+  \hline
+  TrueFxFetcher & the fetcher for  live Forex quotes  \\
+  \hline
+  HistDataCSVFetcher & reads data from csv source and converts it to Quotes \\
+  \hline
+\end{tabularx}\\[0.3cm]
+
+Note that all the fetchers in the table above are not components themselves, except \emph{HistDataCSVFetcher}. Non-component fetchers are used together with \emph{PullFetchComponent} as following snippet shows:
+
+\begin{lstlisting}[language=Scala]
+  val fetcherFx: TrueFxFetcher = new TrueFxFetcher
+  val fetcher = builder.createRef(Props(classOf[PullFetchComponent[Quote]], fetcherFx, implicitly[ClassTag[Quote]]), "TrueFxFetcher")
+\end{lstlisting}
+
+\subsubsection{HistDataCSVFetcher}
+
+This uses historical quote data from the free financial data provider histdata.com\footnote{More specifically http://www.histdata.com/download-free-forex-data/?/ninjatrader/tick-\{ask, bid\}-quotes}. The data we used can be comfortably fetched by using our get-data.sh script, for example like this: \lstinline|scripts/get-data.sh www.histdata.com EURCHF| (gets a set of CSV files to  \lstinline|./data/EURCHF|). HistDataCSVFetcher can take this data and send quotes "replaying" the past. To build a fetcher replaying EUR/CHF data from April 2013 through May 2013 with a speed factor of 60x (i.e. 1 hour is replayed in 1 minute) the following can be done:
```

Comment:
`get-data.sh`: make it use monospace font (I usually use `\texttt{}` but I see `\lstinline` seems to work well also)